cardinal/ecs/read.go

```diff
@@ -57,3 +118,73 @@ func (r *ReadType[req, rep]) HandleRead(w *World, bz []byte) ([]byte, error) {
 	}
 	return bz, nil
 }
+
+func (r *ReadType[req, rep]) DecodeEVMRequest(bz []byte) (any, error) {
+	if r.requestABI == nil {
+		return nil, errors.New("cannot call DecodeEVMRequest without using the WithEVMSupport option when " +
+			"creating the read")
```

Comment:
just wondering: are we using this + in the error for readability?

Reply:
yeah, linter has line length rules